src/main/java/world/bentobox/bentobox/managers/IslandsManager.java

```diff
@@ -1374,7 +1374,8 @@ public void clearArea(Location loc) {
         loc.getWorld().getNearbyEntities(loc, plugin.getSettings().getClearRadius(),
                 plugin.getSettings().getClearRadius(),
                 plugin.getSettings().getClearRadius()).stream()
-        .filter(en -> Util.isHostileEntity(en)
+        .filter(en -> !en.isPersistent()
```

Comment:
This caused a regression. By default (per JavaDocs) all entities are persistent. So actually, this is already false.